### DISCUSSION TITLE
Incompatible mods found

### DISCUSSION THREAD

**Discussion by XeweIs** [FabricMC/fabric-api#4268] [Mod Dev Support] [answered]
### Description

I chose version 1.20.1 at https://fabricmc.net/develop/. I pasted it into gradle.properties and tried to run runClient and got an error that it requires version 1.20.6
![изображение](https://github.com/user-attachments/assets/2d13782b-a90f-46ab-8e76-5689f180fd4f)

**Comment by modmuss50** [comment, marked as answer]
You will need to adjust your fabric.mod.json file for the version of Minecraft you are using. Its located in src/main/resources.

You can also use the online template mod generator to create a new mod for a specific version: https://fabricmc.net/develop/template/
